src/main/java/pl/touk/sputnik/connector/stash/StashFacade.java

```diff
@@ -2,23 +2,16 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.google.common.base.Joiner;
-import com.google.common.collect.Lists;
 import com.jayway.jsonpath.JsonPath;
 import lombok.extern.slf4j.Slf4j;
-import net.minidev.json.JSONObject;
+import net.minidev.json.JSONArray;
 import org.jetbrains.annotations.NotNull;
 import pl.touk.sputnik.configuration.Configuration;
 import pl.touk.sputnik.configuration.GeneralOption;
 import pl.touk.sputnik.configuration.GeneralOptionNotSupportedException;
 import pl.touk.sputnik.connector.ConnectorFacade;
 import pl.touk.sputnik.connector.Connectors;
-import pl.touk.sputnik.connector.stash.json.Anchor;
-import pl.touk.sputnik.connector.stash.json.Comment;
-import pl.touk.sputnik.connector.stash.json.DiffSegment;
-import pl.touk.sputnik.connector.stash.json.FileComment;
-import pl.touk.sputnik.connector.stash.json.LineComment;
-import pl.touk.sputnik.connector.stash.json.LineSegment;
-import pl.touk.sputnik.connector.stash.json.ReviewElement;
+import pl.touk.sputnik.connector.stash.json.*;
```

Comment:
imports without star were better

Reply:
I've optimised all imports and remove all stars! :star2: